docs/tsc/meetings/2019-02-04.md

```diff
@@ -0,0 +1,36 @@
+
+Feb. 4, 2019
+
+Secretary: Kevin Wheatley
+
+Attendees: Kevin Wheatley, John Mertic, Carl Rand, Larry Gritz, Michael “SPI” Dolan, Sean Cooper, Bernard Lefebvre, Doug Walker, Patrick Hodoul, Thanh Ha
+
+# **OCIO TSC Meeting Notes**
+
+* Opening explanation of form and function of TSC by John Mertic.
+* Election of officials/Nominations for roles
+* DW, explained the prior discussion from the Slack #tsc channel
+  * Michael Dolan - Chairperson
+  * Doug Walker for Chief Architect
+* Additional nominations
+  * Sean Cooper - Chairperson
+* Discussion around the concept of Chair + Vice chair and the distinction of only the chair having voting rights on the TAC.
+* LG suggested we may benefit from a singular chair as an efficiency point, rather than the potential of delays by trying to maintain something similar to the current consensus model.
```

Comment:
For external readers, I think that we should use complete names.  Refer to line 18 & line 12